apps/staging-config/package.json

```diff
@@ -12,16 +12,17 @@
     "@grouparoo/core": "0.3.3",
     "@grouparoo/demo": "0.3.3",
     "@grouparoo/postgres": "0.3.3",
-    "@grouparoo/sqlite": "0.3.3"
+    "@grouparoo/sqlite": "0.3.3",
+    "grouparoo": "0.3.3"
```

Comment:
I would advise against this, as `@grouparoo/core` already has `grouparoo` as a dependency.  The demo script could be `./node_module/@grouparoo/core/node_modules/.bin/grouparoo demo purchases --config`

Reply:
@evantahler Making this change then still tries to get into a `cli` directory, which isn't there.

```
Cannot find module '/path/to/monorepo/apps/staging-config/node_modules/@grouparoo/cli/dist/grouparoo.js'
```

Reply:
Fair enough.  There's probably some changes we could make in the demo plugin/CLI loader to work a little better in this case... but that's another story! Carry on.